scss/components/_activity-card.scss

```diff
@@ -199,6 +199,10 @@
   }
 }
 
+.civicase__contact-icon .material-icons {
```

Comment:
Shouldn't this be nested? 
```scss
.civicase__contact-icon {
  .material-icons {
    line-height: inherit;
  }
}
```

Reply:
Also, shouldn't this be the part of `Contact Card` scss component ?

some where here ?
https://github.com/compucorp/uk.co.compucorp.civicase/blob/master/scss/components/_contact-card.scss#L20